kayenta-objectstore-configbin/src/main/java/com/netflix/kayenta/configbin/storage/ConfigBinStorageService.java

```diff
@@ -337,7 +339,7 @@ private Map<String, Object> metadataFor(ConfigBinNamedAccountCredentials credent
                       () -> remoteService.get(ownerApp, configType, id),
                       MAX_RETRIES,
                       RETRY_BACKOFF));
-    } catch (RetrofitError e) {
+    } catch (SpinnakerServerException e) {
       throw new IllegalArgumentException("No such object named " + id);
```

Comment:
Similar question here as https://github.com/spinnaker/kayenta/pull/972/commits/c75541ca7a923355aa651275b6c8b9229b27f1e5#r1240412522, though IllegalArgumentException results in a 400 response, so `e.newInstance` may not work without other changes.

Reply:
That link doesn't take me to a question anymore.  This change seems OK to me as I'm looking at it now.  I do wonder if it would be an improvement to add e as the cause, like
```suggestion
      throw new IllegalArgumentException("No such object named " + id, e);
```